src/eko/evolution_operator/__init__.py

```diff
@@ -603,8 +607,8 @@ class Operator(sv.ModeMixin):
 
     log_label = "Evolution"
     # complete list of possible evolution operators labels
-    full_labels = br.full_labels
-    full_labels_qed = br.full_unified_labels
+    full_labels = list(br.full_labels)
```

Comment:
because it is not of constant size between here and [it's child class](https://github.com/NNPDF/eko/blob/6e0f434b9c1792dad9c79bed9e013e2a5b5cdd1d/src/eko/evolution_operator/operator_matrix_element.py#L199)